context/app/static/js/pages/Publication/Publication.jsx

```diff
@@ -38,11 +39,11 @@ function Publication({ publication, vignette_json }) {
   const shouldDisplaySection = {
     visualizations: Boolean(Object.keys(vignette_json).length),
     provenance: !associatedCollectionUUID,
-    files: true,
+    files: Boolean(files?.length),
```

Comment:
Just for consistency with `Dataset`:

```suggestion
    files: Boolean(files?.length),
    bulkDataTransfer: true
```